### ISSUE TITLE
docs: missing prerequisites on Scanner user/pass and Enforcer enforcerToken

### ISSUE THREAD

**Issue by agilgur5** [aquasecurity/aqua-helm#119]
The docs for Scanner and Enforcer don't mention that they have prerequisites other than an `imagePullSecret`, but both do.

The Scanner chart [requires](https://github.com/aquasecurity/aqua-helm/blob/525c1aae5bfd31b8137fccbbe7353c134163fb14/scanner/templates/scanner-deployment.yaml#L36) that a `user` and `password` are given, but the docs/its README doesn't mention this at all, neither that it is required nor how to create one. Aqua Engineers pointed out to me that this was necessary but I didn't know despite having read through all the docs quite a lot.
I think directions on creating a `user`/`password` should be added in the Scanner README's prerequisites.

The Enforcer chart [requires](https://github.com/aquasecurity/aqua-helm/blob/525c1aae5bfd31b8137fccbbe7353c134163fb14/enforcer/templates/enforcer-token-secret.yaml#L14) that an `enforcerToken` is given, but the docs/its README doesn't mention this at all, neither that it is required nor how to create one. This I noticed from the argument in the Helm install command and knew from the docs site that one must create an Enforcer group, but if one only read here one wouldn't realize this was necessary as it's not mentioned.
I think directions on creating an Enforcer group and getting a `enforcerToken` should be added in the Enforcer README's prerequisites.

## Tags

Mentioned this in https://github.com/aquasecurity/aqua-helm/pull/116

**Comment by KoppulaRajender** [comment]
updated docs, thanks.

**Comment by agilgur5** [comment]
I see #270 added docs for creating a [Scanner User](https://github.com/aquasecurity/aqua-helm/pull/270/files#diff-26f6432cd17508fa13d1fff404ca4e21347f79e1987aab64a1a12071409fe2f3R29), but no docs were added for the Enforcer group and `enforcerToken`, which is the second half of the issue.

I checked the current version, and the [Enforcer docs](https://github.com/aquasecurity/aqua-helm/blame/19dd1539a1d4e48812b5618633099db32d9cb035/enforcer/README.md#L43) still mention nothing about the `enforcerToken`; they just assume you know what that is and how to get it.

@KoppulaRajender as the second half of this issue was never resolved, I believe this issue should be reopened.
I could make another issue, but to be honest, I got so frustrated [here](https://github.com/aquasecurity/aqua-helm/issues?q=involves%3Aagilgur5) that it zapped all my willingness to contribute (notably, this issue was created almost a year and a half ago)
